memory/migrate_pages.py

```diff
@@ -86,6 +93,8 @@ def test(self):
         cmd = './node_move_pages -n %s' % self.nr_chunks
 
         if self.hpage:
+            if (self.dist.name == 'rhel' and self.dist.version >= '9'):
+                self.cancel("Hugepage tests are cancelled on RHEL-9 and later.")
```

Comment:
@kalpanashtty I think this can be handled at line 61-63 itself.
and I feel loginfo is not required there.

Reply:
As replied earlier for huge-page tests we cancel and continue for TPH tests.

Reply:
@Naresh-ibm : Let me know if you need this message to be changed.  
This is primarily an old PR (https://github.com/avocado-framework-tests/avocado-misc-tests/pull/2447), where the submitter left the company so re-sending it. So, you can get the history of the issue from this PR.